vmngclient/primitives/__init__.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING, Optional
+
+from packaging.specifiers import SpecifierSet  # type: ignore
+from packaging.version import Version  # type: ignore
+
+from vmngclient.exceptions import APIVersionException
+
+if TYPE_CHECKING:
+    from vmngclient.response import vManageResponse
+    from vmngclient.session import vManageSession
+
+
+BasePath = "/dataservice"
```

Comment:
We approved with @JimOverholt that we should provide full path instead of using `const` variables

Reply:
@JimOverholt are we sure that there are APIs outside `/dataservice`?
I know about `/j_security_check` which is used for authentication but it definitely is not the part of what we are trying to do here (strictly API endpoints)
OpenAPI spec uses them all without '/dataservice' prefix

Reply:
```python
  "servers": [
    {
      "url": "/dataservice"
    }
```
I like this

Reply:
In my honest opinion its better to provide the whole, instead of part of an endpoint. I know that there are no other APIs than `/dataservice`, but it can be misleading for some users.  @JimOverholt can you call a final decision - should we use const variable `/dataservice` and call methods without it, f.e. `request(url="user/")` or with `request(url="/dataservice/user/")

Reply:
@Jim @tehAgitto Is official api documentation also misleading for You?
![image](https://user-images.githubusercontent.com/116343782/233352142-4295996d-454b-44a2-bd90-0b3b74d11488.png)

Reply:
Let's call it like that then
```suggestion
BASE_PATH: Final[str] = "/dataservice"
```

Reply:
fixed